staging/src/github.com/kubeedge/mapper-framework/_template/mapper/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV GO111MODULE=on \
 
 COPY . .
 
-RUN CGO_ENABLED=0 GOOS=linux go build -o main cmd/main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -tags Nostream -o main cmd/main.go
```

Comment:
Is there any way to control the feature? Like add a startup parameter to control the mdule to start or not, hdyt @WillardHu @luomengY @wbc6080

Reply:
I have considered using a switch to set whether to use this feature, but even if we add a switch, it seems that this part of the code will still be compiled, only not run this part, so the third-party package of stream data processing will still be used. In order to prevent this part of the code from being compiled, what I finally found was the method of conditional compilation. As for other methods, I haven't found them yet.

Reply:
One idea is that handle the logic(determine which files to copy) when the mapper is generated from the template.

Reply:
Oh I see, very good idea, I will try to implement it like this @WillardHu